extension/app/ts/utils/ethereum.ts

```diff
@@ -68,12 +78,6 @@ function isSignedTransaction(maybeSigned: unknown): maybeSigned is ISignedTransa
 		&& 'yParity' in maybeSigned
 }
 
-export function getV(transaction: Pick<ISignedTransaction, 'chainId' | 'yParity'>) {
-	return 'chainId' in transaction && transaction.chainId !== undefined
-	? (transaction.yParity === 'even' ? 0n : 1n) + 35n + 2n * transaction.chainId
-	: transaction.yParity === 'even' ? 27n : 28n
-}
-
```

Comment:
I thought after our discussion we determined that this is necessary?

Reply:
its not as ethers calculates it from yParity so that we don't have to